README.md

```diff
@@ -1,3 +1,5 @@
+Thiis is a for of Jade that adds a new debug option: `debugLines`.
```

Comment:
The issue is that if we merge your pull request, this won't be a fork it will be the official jade repository.